orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/SessionService.java

```diff
@@ -44,7 +44,7 @@ public SessionService(ConfigurationService configurationService) {
     }
 
     public Session createSession() {
-        return new Session(IdGenerator.generate());
+        return new Session(IdGenerator.generate()).withBrowserSessionId(IdGenerator.generate());
```

Comment:
Decided that rather than rewriting the Session interface to have to include a browser session id, it is instead added with this builder pattern where necessary. Makes this PR a smaller change and means it is more easily kept separate and optional, but if browser session ids become used everywhere, perhaps would be better to include in the interface.